apps/evm_family/evm_sign_msg.c

```diff
@@ -431,11 +458,13 @@ void evm_sign_msg(evm_query_t *query) {
   }
 
   if (NULL != sign_msg_ctx.msg_data) {
+    pb_release(EVM_SIGN_TYPED_DATA_STRUCT_FIELDS, &(sign_msg_ctx.typed_data));
```

Comment:
is the pre-condition `if (NULL != sign_msg_ctx.msg_data)` needed for calling `pb_release`?

Reply:
updated